packages/docusaurus-theme-live-codeblock/package.json

```diff
@@ -29,10 +28,12 @@
     "react-live": "2.2.3"
   },
   "devDependencies": {
+    "@docusaurus/core": "2.0.0-beta.9",
```

Comment:
This is a hack so that lerna correctly figures out the dependency graph and builds the core before building those that still implicitly depend on the core's internal APIs (that's still bad, we need to further refactor that)